state/execution.go

```diff
@@ -334,7 +334,7 @@ func execBlockOnProxyApp(
 		return nil, err
 	}
 
-	logger.Info("executed block", "height", block.Height, "num_valid_txs", validTxs, "num_invalid_txs", invalidTxs)
+	logger.Debug("executed block", "height", block.Height, "num_valid_txs", validTxs, "num_invalid_txs", invalidTxs)
```

Comment:
I though this was a relatively important log?

Reply:
sure, will revert. Still fail to see the been fit of having two logs that say similar things right next to each other, but seems we are fine with it.

Reply:
Right I guess we can just have one